bounce/db/club.py

```diff
@@ -47,16 +52,30 @@ def to_dict(self):
         }
 
 
+def can_delete(editor_role):
+    # Only President can delete club
+    if editor_role == 'President':
```

Comment:
Instead of hard-coding strings like `President` you should use the `ROLE` enum you defined above.

Reply:
I found 2 ways to: 
1. Created a new enum class and import the members of the original enum and check like
editor_role ==Role.president 

2.Or use the original enum and check like editor_role == ROLE.enums[0]

I implemented the 1st approach since it's more descriptive. Please let me know if there's a better way.

Reply:
I like the first approach as well